front-end/src/App.js

```diff
@@ -1,25 +1,38 @@
 import './App.css';
 import React from 'react'
-import {BrowserRouter as Router, Switch, Route} from 'react-router-dom'
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
 import Home from './Home.js'
 import TermsOfService from './TermsOfService';
 import MeetTheTeam from './MeetTheTeam';
 import ScrollToTop from './ScrollToTop';
 import Profile from './Profile';
-import MyShows from './MyShows'
+import MyShows from './MyShows';
+import IndividualShow from './IndividualShow';
 import Modal from "react-modal";
 import Login from './Login';
 import Signup from './Signup';
 
+/*
+const IndividualShowWrapper = ({match}) => {
+  return (
+    <IndividualShow id={match.params.id} show_id={match.params.id.show_id}  />
+  )
+}
+
+*/
+
+/*
+<Route path="/my_shows/:id/:show_id" component={IndividualShowWrapper}/>
+*/
```

Comment:
I left it because it might help for anyone to link the individual show page by using user id -- currently the page is linked from the root directory instead of the myshows page

Reply:
Please remove it. We don't need it.